docs/data/migration/migration-data-grid-v4/migration-data-grid-v4.md

```diff
@@ -1,3 +1,7 @@
+---
+productId: x-data-grid
```

Comment:
We probably need to bump monorepo to take advantage of this change. 🤔

Reply:
Am I missing something, I don't seem to notice this `productId` make effect anywhere...
It does not affect the `meta` attribute value, what should it do?

Reply:
Understood, thanks for clarifying. 👌